### PR TITLE
first draft of cli commands

### PR DESCRIPTION
**Pull request by TyDunn**

### CODE REVIEW

File: docs/website/docs/command-line-interface.md [hunk 1]
```diff
@@ -6,4 +6,25 @@ sidebar_position: 7
...
+then run `dlt init` with a [source](./glossary.md#source) name of your choice and one of the three
+[destination](./destinations.md) options. Otherwise, start with the chess.com API to Google BigQuery
+complete example by running `dlt init chess bigquery`.
+
+Follow the [Create a pipeline](./walkthroughs/create-a-pipeline.md) walkthrough to learn more.
+
+## `dlt deploy`
+
+```
+dlt deploy <script>.py github-action --schedule * * * * *
```
Comment (rudolfix) [inline docs/website/docs/command-line-interface.md:23]: the schedule part must be enclosed in quotation marks `"*/30 * * * *"`

Comment (rudolfix) [reply docs/website/docs/command-line-interface.md:23]: t would be good to provide a working example

File: docs/website/docs/command-line-interface.md [hunk 2]
```diff
@@ -6,4 +6,25 @@ sidebar_position: 7
 
 ## `dlt init`
 
-## `dlt deploy`
\ No newline at end of file
+```
+dlt init <source> <destination>
```
Comment (rudolfix) [inline docs/website/docs/command-line-interface.md:10]: t would be good to provide a working example

File: docs/website/docs/command-line-interface.md [hunk 3]
```diff
@@ -6,4 +6,25 @@ sidebar_position: 7
...
+```
+
+If you want to start from a [generic template](https://github.com/dlt-hub/python-dlt-init-template),
```
Comment (rudolfix) [inline docs/website/docs/command-line-interface.md:13]: Following @TyDunn and @adrianbr suggestions we have two generic templates now
1. **debugging template** with `dlt init <source> <destination>` where the data loading code is commented out, instead you have debug prints and mock data returned so you can run the code right away without credentials setup and see how dlt builiding blocks work
2. **generic template** with ` --generic` with all the data loading code present. you need to implement the missing requests to REST apis and provide credentials to run the pipeline for the first time

@rudolfix change the help text in `dlt` to reflect the name of parameters used here

File: docs/website/docs/command-line-interface.md (hunk 1 above)
Comment (rudolfix) [reply docs/website/docs/command-line-interface.md:23]: it is worth stressing that schedule **must** be in quotation marks/apostrophes!

**Review by rudolfix — CHANGES_REQUESTED** [review]

**Comment by TyDunn** [comment]
Merging for now. I'll fix outstanding comments in a new PR

**Pull request merged**